source/Octo/Commands/HealthStatusProvider.cs

```diff
@@ -84,20 +83,20 @@ public IEnumerable<MachineResource> Filter(IEnumerable<MachineResource> machines
 
         IEnumerable<MachineResource> FilterByProvidedStatus(IEnumerable<MachineResource> machines)
         {
-            var statusFilter = new List<MachineModelStatus>();
+            var statusFilter = new List<MachineModelHealthStatus>();
             if (statuses.Count > 0)
             {
                 log.Debug("Loading statuses...");
                 foreach (var status in statuses)
                 {
-                    MachineModelStatus result;
+                    MachineModelHealthStatus result;
                     if (Enum.TryParse(status, true, out result))
                         statusFilter.Add(result);
                 }
             }
 
             return statusFilter.Any()
-                ? machines.Where(p => statusFilter.Contains(p.Status))
+                ? machines.Where(p => statusFilter.Contains(p.HealthStatus))
```

Comment:
So with this change we test either parameters `--healthstatus` or `--status` against `MachineModelHealthStatus`? Shouldnt we just fail now if they try to use the old status, rather than just using it for something else?